storybookjs/storybook
# Story index: Uncache `stories.json` generation

Issue: N/A

## What I did

- [x] Fix `toIndexFn` in `storyStoreV7` mode
- [x] Generate `stories.json` on each request to the stories index

## How to test

```
cd examples/react-ts
yarn storybook
```

Then add or remove files/stories. Refresh the page and the new stories should appear in the nav UI.
